gridlab-d/gridlab-d
# Feature/1239 - Merge constant pf corrections into develop

#### What's this Pull Request do?
Basically addresses #1239, putting in the ability for GridLAB-D to maintain a constant pf for constant current loads as the reference frame rotates.  Previously (non-deltamode), this would need to be done manually by the user.

#### Where should the reviewer start?
Pull the code, build it, and run the autotest script.

#### How should this be tested?
Verify the current rotation occurs properly -- using a zero-referenced delta system and converting back to Wye (to impart a roughly 30 degree shift) is a good way to check this.

New autotests created (test_IEEE_37_XXX.glm and test_constant_current_angle_parents_XXX.glm) to test these aspects, as well as updated to assert values on existing autotests.

#### Any background context you want to provide?
N/A

#### What are the relevant issues?
#1239
